lib/foreman_maintain/cli/base.rb

```diff
@@ -144,7 +144,7 @@ def self.label_option
 
       def self.tags_option
         option('--tags', 'tags',
-               'Limit only for specific set of labels. ' \
+               'Limit only where all specific set of tags exist. ' \
```

Comment:
I still had to read the other BZ to understand the description.
Wouldn't be `Run only checks with all these tags` more explanatory?

To be consistent we could change the description for `--label` to 'Run only check with this label'

@kgaikwad, what do you think?

Reply:
@mbacovsky,  modified description for tags & label.